src/main.js

```diff
@@ -0,0 +1,42 @@
+function strComp (str1, str2) {
+  if (str1.localeCompare(str2) === -1 || str1.localeCompare(str2) === 0){
+    return true;
+  }
+}
+
+function checkAndPush(inputElement, liArray) {
+  const tempLiItem = document.createElement("li");
+  tempLiItem.textContent = inputElement.name;
+  for (j=0; j<liArray.length; j++){
+    if (strComp(inputElement.name.toUpperCase(), liArray[j].textContent.toUpperCase()))
+      break;
+  }
+  return j;
+}
+
+function renderInput(input, container){
+  const dirItem = [];
+  const fileItem = [];
+
+  const dirList = document.createElement("ul");
+  const fileList = document.createElement("ul");
+
+  var i, j;
+  for (i=0; i<input.length; i++) {
```

Comment:
Instead, of using a for loop, let's use the [`forEach` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) instead, which is available in all arrays 😄.

Reply:
Also, the [`map` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) could make a lot of sense here, in order to map array entries to new `li` elements. Example:

```javascript
const entries = [{"name": "blim", "type": "file"}, {"name": "blom", "type": "directory"}];
const elements = entries.map(entry => {
  const element = document.createElement("li");
  element.textContent = entry.name;
  return element;
});
```